specs/protocol/isthmus/configurability.md

```diff
@@ -39,9 +45,44 @@ The following actions should happen during the initialization of the `SystemConf
 - `emit ConfigUpdate.GAS_LIMIT`
 - `emit ConfigUpdate.UNSAFE_BLOCK_SIGNER`
 - `emit ConfigUpdate.EIP_1559_PARAMS`
+- `emit ConfigUpdate.OPERATOR_FEE_PARAMS`
 
 These actions MAY only be triggered if there is a diff to the value.
 
+`ConfigUpdate.OPERATOR_FEE_PARAMS` MAY be emitted. If it is not emitted, the `operatorFeeScalar` and
```

Comment:
I recommend removing this from being called in `initialize` see https://github.com/ethereum-optimism/optimism/pull/12166/files#r1938083725

Reply:
Done